test/AutoDiff/differential_operators.swift

```diff
@@ -6,20 +6,23 @@ import StdlibUnittest
 
 var BuiltinDifferentialOperatorTests = TestSuite("BuiltinDifferentialOperators")
 
-@inlinable
-func valueWithPullback<T, R>(
-  at x: T, in f: @autodiff (T) -> R
-) -> (R, (R.CotangentVector) -> T.CotangentVector)
-  where T : Differentiable, R : Differentiable {
-  return Builtin.autodiffApplyVJP(f, x)
```

Comment:
change `%target-run-simple-parse-stdlib-swift` above, because it no longer requires `-parse-stdlib`

Reply:
Done. I'm keeping it in `lit.cfg` in case you need it for your witness method builtins.